script_hw1.js

```diff
@@ -0,0 +1,59 @@
+// 1
+let user_name = prompt("Enter your name here. Fast:");
+alert(`Hi, ${user_name}!`);
+
+// 2
+const user_birth_year = +prompt("Enter your birth year here:");
+const now_year = 2021;
+alert(`You\`re most likely ${now_year - user_birth_year} old now :(`);
+alert(`If not, you\`ll be ${now_year - user_birth_year} this year!`);
```

Comment:
можно возраст вынести в переменную, чтоб не повторяться
```suggestion
const age = now_year - user_birth_year;
alert(`You\`re most likely ${age} old now :(`);
alert(`If not, you\`ll be ${age} this year!`);
```